### ISSUE TITLE
nation mission 5-1 (all 3 nation) very much not retail

### ISSUE THREAD

**Issue by TeoTwawki** [DarkstarProject/darkstar#918]
All mobs these mobs should not be up and super linked. Only 2 helpers should be up at a time with the archlich. Also a single whm banishaga should not be near a near one hit kill for every skeleton type mob in the game.

**Comment by gedads** [comment]
https://www.youtube.com/watch?v=boEOjTIxfkA
watch near 6:00 up to 7:00
they're not all there at start and are summoned as the fight goes on and come back if defeated. If left alone you can totally get surronded.

**Comment by TeoTwawki** [comment]
Still, right now the second you agro/attack the Archlich, you instantly have ALL of them on you, which is not how retail handles that. Also, the 2 sorcerers never come back, only the 4 warriors. There are 4, but they only pop 2 at a time. So I misspoke saying up at once, its 2 pop at once.

**Comment by Yuchemey** [comment]
Yes I did the fight on era - I also remember that fight that you can't kite the helpers. But I I was able to.
I remember that "a kite" would take a mob even though after a little while it would ve. Loose enmit and the tank would get pwn d

**Comment by maxtherabbit** [comment]
@TeoTwawki is this closable now?

**Comment by TeoTwawki** [comment]
@maxtherabbit I made the skeleton spawns and repops as close to retail as I could figure, but didn't address anything else. 

Did retail severely bump up whm nuke power and we got that or something? I dunno whats up with banish power. Or cure power for that matter: whm have discovered they can near 1 shot IT bones in about every zone for fast exp so was seeing them alternate banishga's and cures to take out groups for quick chains. On legions server they me help them add cure resists to mobs in gusgen so ppl would stop trying to abuse it there with a lv sync alt.
